assets/images/dwolla-developers-logo.svg

```diff
@@ -0,0 +1,16 @@
+  </g>
+</svg>
```

Comment:
Switched to SVG for better image scaling

Reply:
![](https://media0.giphy.com/media/tyxovVLbfZdok/giphy.gif)